metaseq/dataclass/configs.py

```diff
@@ -106,6 +106,17 @@ class CommonConfig(MetaseqDataclass):
             "of running tensorboard (default: no tensorboard logging)"
         },
     )
+    aim_repo: Optional[str] = field(
```

Comment:
Why should this pathing be complete separate from where all other logs go?

Reply:
@suchenzang Aim provides remote tracking functionality, and `aim_repo` can refer a remote server URI rather than local path.